packages/api/src/hooks/useAllAvailableAccounts.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+import useAccountTypes from './useAccountTypes';
+import useLandingCompany from './useLandingCompany';
+
+/** A custom hook to get all available accounts that user can have. */
+const useAllAvailableAccounts = () => {
+    const { data: landing_company_data } = useLandingCompany();
+    const { data: account_types_data, ...rest } = useAccountTypes(landing_company_data?.financial_company?.shortcode);
```

Comment:
@mahdiyeh-deriv We only sending financial_company instead of patching for BE. We create 2 lesser api calls this way which is better 👀